### ISSUE TITLE
Difference between apply() and transform() in Pandas

### ISSUE THREAD

**Issue by rhiriyappa** [rhiriyappa/data-science-articles#3]
# TL;DR

# Article Link

https://towardsdatascience.com/difference-between-apply-and-transform-in-pandas-242e5cf32705

#Key Takeaways

For manipulating values, both **apply()** and **transform()** can be used to manipulate an entire DataFrame or any specific column. 

But there are 3 differences

1. **transform()** can take a function, a string function, a list of functions, and a dict. However, apply() is only allowed a function.
2. **transform()** cannot produce aggregated results
3. **apply()** works with multiple Series at a time. But, transform() is only allowed to work with a single Series at a time.

For working in conjunction with **groupby()**

1. **transform()** returns a Series that has the same length as the input
2. **apply()** works with multiple Series at a time. But, transform() is only allowed to work with a single Series at a time.
3
